packages/runtime/src/remote/index.ts

```diff
@@ -405,6 +418,11 @@ export class RemoteHandler {
   private removeRemote(remote: Remote): void {
     const { host } = this;
     const { name } = remote;
+    for (const [key] of this.loadRemoteCache) {
+      if (key.startsWith(`${name}_`)) {
```

Comment:
the match may match failed , i think it's better to record the name and id for the remote as unique id , and then delete it